plugins/gatsby-source-react-packages/gatsby-node.js

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
Can this `eslint-disable` be localized to the for loop to which it applies?

Reply:
Updated